geoproxy/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Start from the latest golang base image
+FROM golang:latest
+
+# Add Maintainer Info
+LABEL maintainer="Jacob Diaz <cjdiaz98@gmail.com>"
+
+# Set the Current Working Directory inside the container
+WORKDIR /app
+
+# Copy everything from the current directory to the Working Directory inside the container
+COPY . .
+
+# Build the Go app
+RUN go build -o main .
+
+# Expose port 8080 to the outside world
+EXPOSE 8080
+
+# Command to run the executable
+CMD ["./main"]
```

Comment:
Looks good! I was just a little worried about this CMD--If I understand it correctly, we need to make an executable file in this directory called ./main or change the CMD to point to the executable in the bin file.